help/upgrade/upgrade-compatibility-tool/overview.md

```diff
@@ -19,12 +19,12 @@ Refer to the [Install](../upgrade-compatibility-tool/install.md) for the first s
 
 ## Help improve the [!DNL Upgrade Compatibility Tool]
 
-To connect with the [!DNL Upgrade Compatibility Tool] team, contact us on the Engineering Slack channel [[!DNL Upgrade Compatibility Tool]](https://magentocommeng.slack.com/archives/C019Y143U9F).
+To connect with the [!DNL Upgrade Compatibility Tool] team, contact us on the Engineering Slack channel [[!DNL Upgrade Compatibility Tool]](https://magentocommeng.slack.com/archives/C019Y143U9F). We want to hear your feedback, issues, and suggestions to help us improve the tool.
 
->[!NOTE]
->
->We want to hear your feedback, issues, and suggestions to help us improve the tool.
-
-Do you have an idea how to improve our compatibility analysis? Check the [Contribute](https://devdocs.magento.com/guides/v2.4/coding-standards/contributing.html) page for more information.
+The Upgrade Compatibility Tool uses rules defined within our [Coding Standards](https://devdocs.magento.com/guides/v2.4/coding-standards/bk-coding-standards.html), to ensure your project is following Adobe Commerce best practices, and at the same time allows you to improve and extend the Upgrade Compatibility Tool compatibility analysis by contributing to this project. Check the [Contribute](https://devdocs.magento.com/guides/v2.4/coding-standards/contributing.html)  page for more information.
```

Comment:
Don't forget to include the tag, so name is not translated nor localized: [!DNL Upgrade Compatibility Tool].
So:

```
The [!DNL Upgrade Compatibility Tool] uses rules defined within our [Coding Standards](https://devdocs.magento.com/guides/v2.4/coding-standards/bk-coding-standards.html), to ensure your project is following Adobe Commerce best practices, and at the same time allowing you to improve and extend the [!DNL Upgrade Compatibility Tool] compatibility analysis by contributing to this project.

Refer to the [Contribute](https://devdocs.magento.com/guides/v2.4/coding-standards/contributing.html)  topic for more information on contributing to this project.
```